packages/components/scripts/storybook/docs-codepen-enhancer.ts

```diff
@@ -0,0 +1,158 @@
+      if (isEditableStory) {
+        story.querySelectorAll<HTMLElement>('.docblock-codepen-button:not(:last-of-type)').forEach(el => {
+          el.style.display = 'block';
+          el.style.borderRight = 'none';
+        });
+        story.querySelector<HTMLElement>('.docblock-codepen-button:last-of-type')!.style.display = 'none';
+      }
+
+      // Finally add the event listener to the button
+      editCodeButton.addEventListener('click', () => {
+        const form = document.createElement('form');
+        form.action = 'https://codepen.io/pen/define';
+        form.method = 'POST';
+        form.target = '_blank';
+
+        // Docs: https://blog.codepen.io/documentation/prefill/
+        const data = {
+          css: `/* See https://solid-design-system.fe.union-investment.de/${packageVersions['@solid-design-system/components']}/storybook/?path=/docs/docs-general-installation--docs */
+@import url("https://solid-design-system.fe.union-investment.de/${packageVersions['@solid-design-system/components']}/styles/solid-styles.css");
+
+
+/* See https://solid-design-system.fe.union-investment.de/${packageVersions['@solid-design-system/components']}/storybook/?path=/docs/docs-general-prerequisites--docs */
+body {
+  font-family:
+    'Frutiger Neue',
+    ui-sans-serif,
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    Roboto,
+    'Helvetica Neue',
+    Arial,
+    'Noto Sans',
+    sans-serif,
+    'Apple Color Emoji',
+    'Segoe UI Emoji',
+    'Segoe UI Symbol',
+    'Noto Color Emoji';
+}
+
+@font-face {
+  font-family: 'Frutiger Neue';
+  font-style: normal;
+  font-weight: 400;
+  src: url('https: //global-resources.fe.union-investment.de/latest/fonts/frutiger-neue/FrutigerNeuefuerUIWebW05-Bk.woff2')
+    format('woff2');
+}
+
+@font-face {
+  font-family: 'Frutiger Neue';
+  font-style: italic;
+  font-weight: 400;
+  src: url('https: //global-resources.fe.union-investment.de/latest/fonts/frutiger-neue/FrutigerNeuefuerUIWebW05-BkIt.woff2')
+    format('woff2');
+}
+
+@font-face {
+  font-family: 'Frutiger Neue';
+  font-style: normal;
+  font-weight: 600;
+  src: url('https: //global-resources.fe.union-investment.de/latest/fonts/frutiger-neue/FrutigerNeuefuerUIWebW05-Bd.woff2')
+    format('woff2');
+}
+
+@font-face {
+  font-family: 'Frutiger Neue';
+  font-style: italic;
+  font-weight: 600;
+  src: url('https: //global-resources.fe.union-investment.de/latest/fonts/frutiger-neue/FrutigerNeuefuerUIWebW05-BdIt.woff2')
+    format('woff2');
+}
+
+/**
+  * Render Neue Frutiger correctly.
+  */
+
+:root {
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-text-size-adjust: 100%;
+  line-height: 1.5;
+  font-size: 16px;
+}`,
+          css_external: '',
+          description: '',
+          editors: 1110,
+          head: '<meta name="viewport" content="width=device-width">',
+          html: code,
```

Comment:
To improve the formatting in codepen, I suggest removing the empty lines. Something like `code.replace(/\n\s*\n/g, '\n')`